support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -222,22 +222,51 @@ Resources:
       Statistic: Sum
     DependsOn: SupportWorkersPROD
 
-  NoPaypalContributionsInTwoHoursAlarm:
+  NoPayPalContributionsInTwoHoursAlarm:
     Type: AWS::CloudWatch::Alarm
     Condition: CreateProdResources
     Properties:
       AlarmActions:
         - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:contributions-dev
       AlarmName: !Sub support-workers ${Stage} No successful recurring paypal contributions for two hours
-      MetricName: PaymentSuccess
-      Namespace: support-frontend
-      Dimensions:
-        - Name: PaymentProvider
-          Value: PayPal
-        - Name: ProductType
-          Value: Contribution
-        - Name: Stage
-          Value: !Ref Stage
+      Metrics:
+        - Id: e1
+          Expression: "SUM([FILL(m1,0),FILL(m2,0)])"
+          Label: AllRecurringPayPalContributions
+        - Id: m1
+          Label: String
+          MetricStat:
+            Metric:
+              Dimensions:
+                - Name: PaymentProvider
+                  Value: PayPal
+                - Name: ProductType
+                  Value: Contribution
+                - Name: Stage
+                  Value: !Ref Stage
+              MetricName: PaymentSuccess
+              Namespace: support-frontend
+            Period: 300
+            Stat: Sum
+            Unit: Count
+          ReturnData: false
+        - Id: m2
```

Comment:
So what is happening here is that we're adding a new metric to alarm on with a different `ProductType` dimension, is that right?

Reply:
The intention is to have the alarms that currently fire when there are no recurring contributions also count SupporterPlus – i.e. they should only alarm if there are no recurring contributions _and_ no SupporterPlus checkouts.